flask-redis/README.md

```diff
@@ -0,0 +1,81 @@
+## Compose sample application
+
+### Python/Flask application using a Redis database
+
+Project structure:
+
+```
+.
+├── Dockerfile
+├── README.md
+├── app.py
+├── docker-compose.yml
+└── requirements.txt
+```
+
+[_docker-compose.yml_](docker-compose.yml)
+
+```
+services:
+   redis: 
+     image: redislabs/redismod
+     ports:
+       - '6379:6379' 
+   web:
+        build: .
+        ports:
+            - "5000:5000"
+        volumes:
+            - .:/code
+        depends_on:
+            - redis
+```
+
+## Deploy with docker-compose
+
+```
+$ docker-compose up -d
+Creating network "flask-redis_default" with the default driver
+Pulling redis (redislabs/redismod:)...
+...
+...
+web_1    |    WARNING: This is a development server. Do not use it in a production deployment.
+web_1    |  * Running on http://172.19.0.3:5000/ (Press CTRL+C to quit)
+web_1    |  * Restarting with stat
+web_1    |  * Debugger is active!
+web_1    |  * Debugger PIN: 598-320-965
+```
```

Comment:
Seems to be the output of a `docker compose up` command not a detached one, it can be disturbing for new comers to not have the same output as described in the README.md